judge/models/problem.py

```diff
@@ -525,3 +544,32 @@ class Meta:
         )
         verbose_name = _('solution')
         verbose_name_plural = _('solutions')
+
+
+class ProblemPointsVote(models.Model):
+    points = models.IntegerField(
+        verbose_name=_('how much this vote is worth'),
+        help_text=_('The amount of points you think this problem deserves.'),
+        validators=[
+            MinValueValidator(settings.DMOJ_PROBLEM_MIN_USER_POINTS_VOTE),
+            MaxValueValidator(settings.DMOJ_PROBLEM_MAX_USER_POINTS_VOTE),
+        ],
```

Comment:
No, it is for form validation.

Reply:
What happens when this is _higher_ than what is specified in the database

Reply:
If form validation is doing our validation, should the database do any?

Reply:
I think (again, this is just going from memory) that this the server-side validation (relying on client-side validation is...bad).

It will throw a `ValidationError` upon a mismatch (what this means in practice is the form submission will be refused, and unless you're doing custom handling for this, nothing should commit die).

This isn't a "mismatch" from the migration, I suspect that this isn't actually written to DB, but is rather another Django construct (and for the migration purposes it probably just resolves down the constant).

Reply:
You're saying the migration throws away the constants it's given?

Reply:
I'm just going to say "assuming they auto-generated the migration with Django (as they should) I think this is fine because this is probably just a Django construct and the actual values in the migration are irrelevant"

Reply:
uh so do we have to do anything here?